pyopensprinkler/station.py

```diff
@@ -48,13 +82,63 @@ def seconds_remaining(self):
         """Retrieve seconds remaining"""
         return self._get_status_variable(1)
 
+    @property
+    def start_time(self):
+        """Retrieve start time"""
+        return self._get_status_variable(2)
+
+    @property
+    def max_name_length(self):
+        return self._controller._state["stations"]["maxlen"]
+
+    @property
+    def masop(self):
```

Comment:
with these properties should we have a more user readable names?

Reply:
I'm game. Finishing up a similar set for programs right now..

Reply:
Want to review the applicable properties for both stations and programs and propose names?